applications/nrf5340_audio/src/bluetooth/Kconfig.defaults

```diff
@@ -48,7 +48,7 @@ config BT_AUTO_DATA_LEN_UPDATE
 	default n
 
 config BT_L2CAP_TX_BUF_COUNT
-	default 6
```

Comment:
This will have to increase later probably, when we will add more services and profiles, so I don't see the need to have this as low as possible. Of course we don't need to overboard with TX buf counts, but the changes in image size isn't much.